_content/essays/advice-to-myself_chokyi-lodro.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Advice to Myself Exposing Hidden Flaws"
+authors:
+  - chokyi-lodro
+subcat: poetry
+translator: "Adam Pearcey"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/jamyang-khyentse-chokyi-lodro/advice-to-self-song"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1jLlmueSREJ0uSVuoHBLrikgIzeGSp1H6/view?usp=sharing"
+course: problems
+tags:
+  - refuge
+  - lamentation
+  - guru-worship
+  - death
```

Comment:
death is a subtopic of problems so not necessary I think. Perhaps `thought` instead?